lib/internal/shutil.js

```diff
@@ -0,0 +1,310 @@
+'use strict';
```

Comment:
ISC license and copyright is missing, isn't it?

> It's ISC for a reason, so you of course already have permission to take the code, as long as the copyright and permission stick around. If it's easier for Node.js to relicense it as MIT, that's probably doable.

- https://github.com/iansu/node/pull/1#issuecomment-489389946

Reply:
I wasn't sure about the correct way to incorporate them so that's why they're missing. It's on the todo list though.

Reply:
I think it should be enough to add it to `LICENSE` [like this](https://github.com/nodejs/node/blob/b6148402d461ffb70446ddd4b55051e093ad1aef/LICENSE#L1472), no need to litter files with licencing comments.

Reply:
if we are going to vendor rimraf could we do it similarly to how we've vendored other modules. Please look at the experimental fetch PR for an example of how to do this (including the license)

https://github.com/nodejs/node/pull/27979